airflow/models/__init__.py

```diff
@@ -4050,8 +4164,14 @@ def sync_to_db(self, owner=None, sync_time=None, session=None):
         if not orm_dag:
             orm_dag = DagModel(dag_id=self.dag_id)
             self.log.info("Creating ORM DAG for %s", self.dag_id)
-        orm_dag.fileloc = self.fileloc
-        orm_dag.is_subdag = self.is_subdag
+        if self.is_subdag:
+            orm_dag.parent_dag = self.parent_dag.dag_id
+            orm_dag.is_subdag = True
```

Comment:
kinda confused. I saw you remove this attribute?

Reply:
This is inside the sync_to_db method, means this is juist copies the value from DAG to DagModel